apps/spruce/src/pages/waterfall/InactiveVersions/InactiveVersionsModal.tsx

```diff
@@ -29,6 +34,7 @@ export const InactiveVersionsModal: React.FC<Props> = ({
       {versions?.map((version) => (
         <StyledVersionLabel
           key={version.id}
+          highlighted={revision !== null && version.revision.includes(revision)}
```

Comment:
This is a nice addition! Do you think it would make sense to apply the highlight around the inactive commits button to provide an indicator that the commit is found there?

Reply:
Great idea! I came up with this glowing effect like the one we use in Parsley. 
![image](https://github.com/user-attachments/assets/03b8ccda-6300-440c-ab5e-c56c940695d0)